project_app/components/styled.tsx

```diff
@@ -0,0 +1,84 @@
+import styled from "styled-components";
+
+export const AppBar = styled.div`
+position: absolute;
+width: 1440px;
+height: 700px;
+left: -4px;
+top: 0px;
+background: #353535;
+`
```

Comment:
lets apply prettier with husky so indentations will be handled

Reply:
installed

Reply:
i dont see anywhere in package.json or a .husky folder